sources/wp-rest-cache.php

```diff
@@ -23,7 +23,11 @@
 require_once ABSPATH . 'wp-admin/includes/plugin.php';
 if ( is_plugin_active( 'wp-rest-cache/wp-rest-cache.php' ) ) {
 	include_once WP_PLUGIN_DIR . '/wp-rest-cache/wp-rest-cache.php';
-
-	$wp_rest_cache_api = new \WP_Rest_Cache_Plugin\Includes\API\Endpoint_Api();
-	$wp_rest_cache_api->get_api_cache();
+} else if ( file_exists( WPMU_PLUGIN_DIR . '/wp-rest-cache/wp-rest-cache.php' ) ) {
```

Comment:
:warning: **Warning**: Usage of ELSE IF is discouraged; use ELSEIF instead (*PSR2.ControlStructures.ElseIfDeclaration.NotAllowed*).